doc/develop/test/twister.rst

```diff
@@ -417,6 +417,8 @@ harness: <string>
     - test
     - console
     - pytest
+    - gtest: Use this harness if you've already got tests written in the gTest
+      framework and do not wish to update them to zTest.
```

Comment:
the previous paragraph says: " The console harness tells Twister to parse a test's text output for a regex defined in the test's YAML file.". Maybe instead of having the comment for gtest here, where harnesses are listed, move it to a new paragraph, just after the list. Also move the sentence from before. E.g.

`Harnesses ztest, gtest and console are based on parsing of the output and matching certain phrases. Ztest and gtests look for pass/fail/etc. frames defined in those frameworks. The console harness tells Twister to parse a test's text output for a regex defined in the test's YAML file.` ?

Reply:
That's way better than anything I've come up with, thanks!